docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.7"
+services:
+  nginx:
+    build:
+      context: infra/nginx
+      dockerfile: Dockerfile
+    container_name: nginx
+    ports:
+      - target: 80
+        published: 8080
+        protocol: tcp
+        mode: host
+    links:
+      - vvgo
+      - minio
+    networks:
+      - frontend
+      - backend
+
+  vvgo:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: vvgo
+    environment:
+      MINIO_ENDPOINT: minio:9000
+      MINIO_ACCESS_KEY_ID: minioadmin
+      MINIO_SECRET_ACCESS_KEY: minioadmin
+    expose:
+      - 8080
+    links:
+      - minio
```

Comment:
Aren't links deprecated and/or discouraged? Bit rusty on Docker networking.

Reply:
I was just following the docs /shrug

Reply:
Yeah, they're considered legacy.

- https://docs.docker.com/network/links/
- https://docs.docker.com/network/bridge/#differences-between-user-defined-bridges-and-the-default-bridge

Reply:
>networks is optional and indicates to which networks containers should be connected.
(By default, containers will be connected on a private, per-compose-file network.)

Looks like we don't need to define any networks for this at all!